src/components/Avatar/index.js

```diff
@@ -17,7 +17,7 @@ function defaultImageFetcher({ onLoad, onError, src }) {
   const img = new Image();
   img.referrerPolicy = 'no-referrer';
   img.crossOrigin = 'Anonymous';
-  img.onload = function() {
+  img.onload = function () {
```

Comment:
I don't really care about `function()` vs. `function ()`, and it seems to be purely stylistic, not avoiding possible errors.

In that case, the question is: why the change? Are we opting into a new version of recommendations as a byproduct of this PR?

Reply:
I guess my rule of thumb is: shorter is better, if it's not a legibility or avoiding bug issue.
But if this is blessed by the folks who do eslint recommendation presets, I give them a little extra benefit of the doubt.

Reply:
Ah! This is probably a change in eslint defaults? I don't care for it either

Reply:
Turns out Prettier 2.0 forces spaces between `function` and the `()`s. Reverted to 1.19